vignettes/gettingStarted.Rmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "Getting Started"
+output: rmarkdown::html_vignette
+---
+
+### Installing:
+```R
+library("devtools")
+install_github("primitybio/cellengine-r-toolkit")
+```
+
+### First steps
+#### Authentication:
+It is recommended that you set your password in your
+`.Renviron`. You can also pass your password as text, but remember that this is
+stored by `R`.
+```R
+library("cellengine")
+authenticate("username", Sys.getenv("CELLENGINE_PASSWORD"))
+```
+
+#### Get experiments:
+
+Experiments may be listed from your account, or requested individually by ID or
+name.
+```R
+experiments = getExperiments()
+exp1 = experiments[1,]
+exp2 = getExperiment("57c4d7d4e69a1e890c4d8d2f")
+exp3 = getExperiment(byName("experiment_name"))
+```
+
+
+### Looking up Resources by Name:
+
+Gates, Populations, FcsFiles, and Compensations can all be requested by name.
+If you are looking up a resource from a specific experiment, this must be done
+with the `byName` function:
+```R
+gate = getGate("57c4d7d4e69a1e890c4d8d2f", byName("gate_name"))
```

Comment:
For clarity can you make this `experimentId = "57c4d7d4e69a1e890c4d8d2f"` `gate = getGate(experimentId, ...)` please?